### PR TITLE
[e2e tests]: Use FIPS compliant cipher in crypto cfg tests

### PR DESCRIPTION
**Pull request by akalenyu**
Signed-off-by: Alex Kalenyuk <akalenyu@redhat.com>

**What this PR does / why we need it**:
Testing on FIPS-compliant clusters might fail miserably because the cipher we pick for testing purposes is not FIPS-friendly.
https://wiki.openssl.org/index.php/FIPS_mode_and_TLS

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
It seems the failure would occur on the top level afterEach so can't even clean up the cipher field,
potentially messing up an entire test suite

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by akalenyu** [comment]
/hold

we have cleanup in ```AfterEach ->
TestCleanup -> 
resetToDefaultConfig```
it's just failing

File: tests/infra_test.go [hunk 1]
```diff
@@ -392,6 +394,10 @@ var _ = Describe("[Serial][sig-compute]Infrastructure", func() {
 
 		})
 
+		AfterEach(func() {
```
Comment (rmohr) [inline tests/infra_test.go:397]: There is a global AfterEach which should take care of that:

https://github.com/kubevirt/kubevirt/blob/main/tests/tests_suite_test.go#L100

and

https://github.com/kubevirt/kubevirt/blob/0f2c6714ae95371f14b34361f747c2311df18a4d/tests/utils.go#L134

**Comment by xpivarc** [comment]
/test pull-kubevirt-e2e-k8s-1.24-sig-compute 
Thanks!
/approve

File: tests/infra_test.go (hunk 1 above)
Comment (akalenyu) [reply tests/infra_test.go]: Yup. thx

**Comment by akalenyu** [comment]
/test pull-kubevirt-e2e-k8s-1.24-sig-compute

with some ciphers (like the one I initially picked) I see that nmap is reporting TLS1.3 is the only available protocol:
```bash
# nmap --script ssl-enum-ciphers -p 8440 127.0.0.1
Starting Nmap 7.92 ( https://nmap.org ) at 2022-09-21 15:03 IDT
Nmap scan report for localhost (127.0.0.1)
Host is up (0.00012s latency).

PORT     STATE SERVICE
8440/tcp open  unknown
| ssl-enum-ciphers: 
|   TLSv1.3: 
|     ciphers: 
|       TLS_AKE_WITH_AES_128_GCM_SHA256 (ecdh_x25519) - A
|       TLS_AKE_WITH_AES_256_GCM_SHA384 (ecdh_x25519) - A
|       TLS_AKE_WITH_CHACHA20_POLY1305_SHA256 (ecdh_x25519) - A
|     cipher preference: server
|_  least strength: A

Nmap done: 1 IP address (1 host up) scanned in 0.26 seconds
```

Not sure why that is or if its kubevirt related. Will consider opening an issue,
for now, picked another FIPS-compliant cipher.

**Review by acardace — APPROVED** [review]

**Pull request merged**